e2e/nx-plugin/src/nx-plugin.test.ts

```diff
@@ -268,8 +273,17 @@ describe('Nx Plugin', () => {
         return JSON.stringify(project, null, 2);
       });
 
+      const graph = readDependencyGraph();
+
       expect(() => checkFilesExist(`libs/${generatedProject}`)).not.toThrow();
       expect(() => runCLI(`execute ${generatedProject}`)).not.toThrow();
+      expect(
```

Comment:
Shouldn't the expectation be along the lines of something shouldn't be cached? instead of reaching into the project graph. Ultimately, we need the graph to be correct so that the cache and affected work right?

Reply:
Hmmm, yeah. That makes more sense. I can update the tests.